.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,29 @@
+name: "Chromatic"
+on: push
+jobs:
+  chromatic:
+    name: Run Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
```

Comment:
this will run on changes to every branch ?

Reply:
Yes, @vikrantgupta25, this will run for every branch and publish storybook for changes to the branch. IMO, it's going to be helpful as it makes it easier to see a preview of the changes in a branch. 

Ref: https://www.chromatic.com/docs/permalinks/

Reply:
what happens for the forks of the repo for OSS contributions ?

Reply:
as I checked (by forking the repo and raising a PR from the forked repo (https://github.com/SigNoz/components/pull/24)), it doesn't run the pipeline. 

I think there might be some option in Repo Settings -> Github Actions to allow running the pipeline for PRs from forked repos.

CC: @YounixM

Reply:
It should not run on fork repos. check this - https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/

Reply:
approving as discussed as we are not running on external PRs for now.